src/services/item/plugins/validation/processes/badWordsDetection.ts

```diff
@@ -9,19 +9,20 @@ export const buildWordList = (badWordsFilter: BadWordsFilter): void => {
 };
 
 export const detectFieldNameWithBadWords = (
-  documents: {
+  _documents: {
     name: string;
     value: string;
   }[],
 ): string[] => {
-  const contents = documents?.filter(Boolean);
-  const badWordsFilter = new BadWordsFilter();
-  buildWordList(badWordsFilter);
-  const suspiciousFields: string[] = [];
-  for (const { value, name } of contents) {
-    if (badWordsFilter.isProfane(value)) {
-      suspiciousFields.push(name);
-    }
-  }
-  return suspiciousFields;
+  return [];
+  // const contents = documents?.filter(Boolean);
+  // const badWordsFilter = new BadWordsFilter();
+  // buildWordList(badWordsFilter);
+  // const suspiciousFields: string[] = [];
+  // for (const { value, name } of contents) {
+  //   if (badWordsFilter.isProfane(value)) {
+  //     suspiciousFields.push(name);
+  //   }
+  // }
+  // return suspiciousFields;
```

Comment:
As precaution, you could leave a comment to remind us in the future why there is this commented code here? Maybe with a link to the related issue.

Reply:
Yes I will add more detail to explain why this is commented out and possible solutions.